ex_3.js

```diff
@@ -0,0 +1,44 @@
+//Folosind instrucțiunea JavaScript switch se va obține ziua săptămânii
+
+//Veți crea o funcție care va returna un șir de caractere (string) care va reprezenta numele unei zile a săptămânii în raport cu numărul acesteia pe parcursul unei săptămâni.
+
+//Exemplu 1:
+
+//Input: 1
+
+//Output: “Luni”
+
+//Exemplu 2:
+
+//Input: 5
+
+//Output: “Vineri”
+
+function week_day(numberday) {
+
+    switch (numberday) {
+        case 0:
```

Comment:
```suggestion
        case 0, 7:
```

7 poate fi si el duminica